gnome-image-installer/gnome-image-installer.c

```diff
@@ -162,7 +162,7 @@ static gboolean
 check_for_live_boot (gchar **uuid)
 {
   const gchar *force = NULL;
-  GError *error = NULL;
+  g_autoptr(GError) error = NULL;
```

Comment:
Nitpick: I’ve been trying to be more consistent at using `local_error` as the name for a `GError*` local variable, and `error` as the name for a `GError**` parameter, since then you can add a `GError **error` parameter to an existing function without having to s/`error`/`local_error`/ its local variable first. However, what you have here is definitely correct, so this is more a comment for future reference.